### PR TITLE
Added couple of RAID controller models

### PR DESCRIPTION
**Pull request by staralex86**

### CODE REVIEW

File: pci.ids [hunk 1]
```diff
@@ -88,6 +88,12 @@
 		1000 30e0  SAS9300-8i
 	00c9  SAS3216 PCI-Express Fusion-MPT SAS-3
```
Comment (alatarum) [inline pci.ids]: extra empty line

File: pci.ids [hunk 2]
```diff
@@ -96,6 +102,7 @@
 	51a3  7300 Dual Port U.2 MAX
 144d  Samsung Electronics Co Ltd
 	a808  NVMe SSD Controller SM981/PM981/PM983
+	a80a  NVMe SSD Controller PM9A1/980PRO
 		1d49 403b  Thinksystem U.2 PM983 NVMe SSD
```
Comment (alatarum) [inline pci.ids:104]: now "1d49 403b  Thinksystem U.2 PM983 NVMe SSD" become subsystem of a80a instead of a808. Is it intentional?

Comment (staralex86) [reply pci.ids:104]: It's my mistake, need make "1d49 403b  Thinksystem U.2 PM983 NVMe SSD" subsystem of a808

File: pci.ids [hunk 3]
```diff
@@ -57,7 +57,7 @@
 
 # Accelerators
 #------------
-
+1002  Advanced Micro Devices, Inc. [AMD/ATI]
```
Comment (AlexanderAmelkin) [inline pci.ids]: Maybe add some devices here? I don't like empty manufacturers in the list of validated equipment.

Comment (AlexanderAmelkin) [reply pci.ids]: Removed this and fixed the rest.

**Review by AlexanderAmelkin — APPROVED** [review]

**Pull request merged**